File: Practice/21/Python/21th.py
def BMI(weight,height):
    return weight / (height**2)

def printBMI(BMI):
    if   BMI < 18.5:
        print('Ниже нормального веса')
    elif BMI < 25:
        print('Нормальный вес')
    elif BMI < 30:
        print('Ожирение I степени')
    elif BMI < 40:
        print('Ожирение II степени')
    else:
        print('Ожирение III степени')

weight = float(input())
height = float(input())
printBMI(BMI(weight, height/100))

File: Practice/21/C++/21th/21th/21th.cpp
#include <iostream>
double BMI(double weight, double height);
void printBMI(double BMI);
int main()
{
    setlocale(LC_ALL, "Russian");
	double weight, height;
	std::cout << "Введите рост(в сантиметрах) и вес(в килограммах)\n";
	std::cin >> height >> weight;
	printBMI(BMI(weight, height/100));
}
double BMI(double weight, double height) {
	return weight / (height * height);
}
void printBMI(double BMI) {
	if (BMI < 18.5)
		std::cout << "Ниже нормального веса";
	else if (BMI < 25)
		std::cout << "Нормальный вес";
	else if (BMI < 30)
		std::cout << "Ожирение I степени";
	else if (BMI < 40)
		std::cout << "Ожирение II степени";
	else
		std::cout << "Ожирение III степени";
}